dependency-manager-tests/cocoapods/CPProject/ViewController.swift

```diff
@@ -7,7 +7,9 @@
 import UIKit
 import Datadog
 
-class ViewController: UIViewController {
+internal class ViewController: UIViewController {
+    private var logger: Logger! // swiftlint:disable:this implicitly_unwrapped_optional
```

Comment:
why do we need an instance prop?

Reply:
No reason 🤷‍♂, this will work anyway even if `Logger` is allocated only for the current stack.